library/src/image.cxx

```diff
@@ -208,11 +246,30 @@ bool image::operator!=(const image& reference) const
 }
 
 //----------------------------------------------------------------------------
-void image::save(const std::string& path) const
+void image::save(const std::string& path, Format format) const
 {
   auto importer = this->Internals->GetVTKImporter();
 
-  vtkNew<vtkPNGWriter> writer;
+  vtkSmartPointer<vtkImageWriter> writer;
```

Comment:
no vtkImageWriterFactory I suppose ?

Reply:
no way to use the image exporter here ?

Reply:
Nop, no factory. The `vtkExporter` is used to get the data from a `vtkImageData`, I don't see how it can be useful.